site/docs/reference/Connectors/capture-connectors/amazon-s3.md

```diff
@@ -0,0 +1,81 @@
+See the [AWS blog](https://aws.amazon.com/blogs/security/wheres-my-secret-access-key/) for help finding these credentials.
+
+## Configuration
+
+There are various ways to configure and implement connectors. See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about these methods. The values and code sample in this section provide configuration details specific to the Amazon S3 source connector.
+
+:::tip
+You might organize your S3 bucket using [prefixes](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-prefixes.html) to emulate a directory structure.
+This connector can use prefixes in various ways, giving you precise control over how datasets are captured into Flow.
+You can specify a prefix in the endpoint configuration to limit the overall scope of the capture within your bucket.
+You'll also specify prefixes on a per-binding basis, allowing you to map each prefix to a distinct Flow collection.
+
+Alternatively, you can capture the entire bucket by omitting `prefix` in the endpoint configuration and
+setting `stream` to the name of the bucket.
+:::
+
+### Values
+
+#### Endpoint
+
+| Value | Name| Description | Type | Required/Default |
+|---|---|---|---|---|
+| `ascendingKeys`| Ascending Keys | Improve sync speeds by listing files from the end of the last sync, rather than listing the entire bucket prefix.* | boolean | false |
+| `awsAccessKeyId`| AWS Access Key ID | Part of the AWS credentials used to connect to S3. Required unless the bucket is public. | string | `"example-aws-access-key-id"` |
+| `awsSecretAccessKey` | AWS Secret Access Key | Part of the AWS credentials used to connect to S3. Required unless the bucket is public. | string | `"example-aws-secret-access-key"` |
+| `bucket` | Bucket | Name of the S3 bucket. | string | Required |
+| `endpoint`| AWS Endpoint | The AWS endpoint URI to connect to, useful if you're capturing from a S3-compatible API that isn't provided by AWS. | string | |
+| `matchKeys` | Match Keys | Regex filter applied to all object keys under the prefix. Only objects whose absolute path match are read. For example, the match key `".*\\.json\"` captures only JSON files. | string | |
+| `prefix` | Prefix | Prefix within the bucket to capture from. | string | |
+| `region` | AWS Region | The name of the AWS region where the S3 bucket is located. | string | `"us-east-1"`, Required |
+
+*To use ascending keys, you must write objects in ascending lexicographic order, such as using RFC-3339 timestamps to record modification times.
+This ensures that key ordering matches the order of changes.
+
+#### Bindings
+
+| Value | Name| Description | Type | Required/Default |
+|---|---|---|---|---|
+| `stream` | Prefix | Path to dataset in the bucket, formatted as `bucket-name/prefix-name` | string | Required |
+| `syncMode` | Sync mode | Connection method. Always set to `incremental`. | string | Required |
```

Comment:
Doesn't have to be a part of this PR, but I think we should talk about `syncMode`. It seems like we should maybe just omit this from all of our docs, and we should maybe also consider whether we can remove it from the catalog models, too.

Reply:
Definitely worth talking about. I didn't have tables for bindings initially, but then I added them because it was raising questions for me. (what's required of a binding? Are they standard or are they defined by connectors? What do these properties _mean_? Etc). Whenever something like that happens, I always assume users will eventually arrive at the same questions and so I add more detail. 
If it's something that a user will see - especially something that's required - it could potentially cause trouble if we never talk about it. Even if it's a vestigial property from Airbyte. 
Removing it from the catalog models, on the other hand, sounds rather appealing. :) Although I'm sure it has plenty of implications.